components/scream/src/physics/spa/tests/CMakeLists.txt

```diff
@@ -14,5 +14,13 @@ CreateUnitTest(spa_one_to_one_remap_test "spa_one_to_one_remap_test.cpp" "${NEED
   LABELS "spa"
   MPI_RANKS 1 ${SCREAM_TEST_MAX_RANKS}
 )
+CreateUnitTest(spa_main_test "spa_main_test.cpp" "${NEED_LIBS}"
```

Comment:
The operations in spa_main are all rank-local, right? Is there any feature that is tested by running with 2+ ranks? If not, I would remove the multi-rank tests.

Reply:
Good point, the reading of data should be tested on multiple ranks, but that is handled by it's own unit test, so I can delete this one running on multiple ranks.